Sources/Swiftly/Init.swift

```diff
@@ -0,0 +1,206 @@
+import ArgumentParser
+import Foundation
+import SwiftlyCore
+
+internal struct Init: SwiftlyCommand {
+    public static var configuration = CommandConfiguration(
+        abstract: "Perform swiftly initialization into your user account."
+    )
+
+    @Flag(name: [.customShort("n"), .long], help: "Do not attempt to modify the profile file to set environment variables (e.g. PATH) on login.")
+    var noModifyProfile: Bool = false
+    @Flag(name: .shortAndLong, help: """
+    Overwrite the existing swiftly installation found at the configured SWIFTLY_HOME, if any. If this option is unspecified and an existing \
+    installation is found, the swiftly executable will be updated, but the rest of the installation will not be modified.
+    """)
+    var overwrite: Bool = false
+    @Option(name: .long, help: "Specify the current Linux platform for swiftly.")
+    var platform: String?
+
+    @OptionGroup var root: GlobalOptions
+
+    public mutating func validate() throws {}
+
+    internal mutating func run() async throws {
+        try await Self.execute(assumeYes: self.root.assumeYes, noModifyProfile: self.noModifyProfile, overwrite: self.overwrite, platform: self.platform)
+    }
+
+    /// Initialize the installation of swiftly.
+    internal static func execute(assumeYes: Bool, noModifyProfile: Bool, overwrite: Bool, platform: String?) async throws {
+        try Swiftly.currentPlatform.verifySwiftlySystemPrerequisites()
+
+        let config = try? Config.load()
+
+        if let config = config, !overwrite && config.version != SwiftlyCore.version {
+            // We don't support downgrades, and we don't yet support upgrades
+            throw Error(message: "An existing swiftly installation was detected. You can try overwriting it.")
+        }
+
+        // Give the user the prompt and the choice to abort at this point.
+        if !assumeYes {
+            SwiftlyCore.print("""
+            Swiftly will be installed into the following locations:
+
+            \(Swiftly.currentPlatform.swiftlyHomeDir.path) - Data and configuration files directory including toolchains
+            \(Swiftly.currentPlatform.swiftlyBinDir.path) - Executables installation directory
+
+            Note that the locations can be changed with SWIFTLY_HOME and SWIFTLY_BIN environment variables and run
+            this again.
+
+            Proceed with the installation?
```

Comment:
Can we change this to a [Y/n]?

Reply:
I've changed the prompt to a yes or no instead of numbered options.